server_config.cfg

```diff
@@ -383,6 +418,17 @@ el_car_class_restriction=a
 el_car_restriction=
 el_weather=
 
+#pinehills raceway rally circuit
+el_add=mixed3_r3_rev
+el_gamemode=racing
+el_laps=4
+el_bots=
+el_car_reset_disabled=0
+el_wrong_way_limiter_disabled=1
+el_car_class_restriction=a
+el_car_restriction=
+el_weather=
```

Comment:
why add this if not using this?

Reply:
not sure what you mean? I added this track to the track list. it's not a duplicate and it is not disabled. and the format is the same as the rest of the tracks

Reply:
I don't see it being added to the loop. 
<img width="480" alt="image" src="https://github.com/user-attachments/assets/7f8fa7d6-23a3-4564-858b-8b6487445eb5">

Reply:
the screenshotted section was a slight re-order of the tracks. I wanted to place sandstone a bit farther away from hellride so it'll potentially be skipped less. So the track list went from this:
..., deathloop, motorcity, devil's canyon, sandstone, hellride, ...
to this:
..., deathloop, sandstone, motorcity, devil's canyon, hellride, ...